packages/core/src/runtime/resolveStyleRules.test.ts

```diff
@@ -99,7 +122,6 @@ describe('resolveStyleRules', () => {
           '--fooBar': 'var(--barBaz)',
 
           backgroundColor: 'red',
-          MozAnimation: 'initial',
```

Comment:
Let's use another vendor property for this test.

Reply:
used `MozBorderLeftColors`